Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -387,6 +446,16 @@ extension SearchSettingsTableViewController {
         model.shouldShowSearchSuggestions = toggle.isOn
     }
 
+    @objc
+    func didToggleEnableFirefoxSuggestions(_ toggle: ThemedSwitch) {
```

Comment:
```suggestion
    func didToggleEnableNonSponsoredSuggestions(_ toggle: ThemedSwitch) {
```

Same naming suggestion here as for the others—`EnableFirefoxSuggestions` could be potentially confused with "Firefox Suggest", while this switch is really referring to non-sponsored suggestions.